samples/cpp/benchmark_app/main.cpp

```diff
@@ -437,6 +437,9 @@ int main(int argc, char* argv[]) {
             compiledModel = core.compile_model(FLAGS_m, device_name);
             auto duration_ms = get_duration_ms_till_now(startTime);
             slog::info << "Load network took " << double_to_string(duration_ms) << " ms" << slog::endl;
+            slog::info << "Original network I/O parameters:" << slog::endl;
+            printInputAndOutputsInfoShort(compiledModel);
```

Comment:
It seems it a part of another PR

Reply:
There were merging issues, now it is fixed